### PR TITLE
Refactor add delete breakpoint debugger extension api

### PR DESCRIPTION
**Pull request by KushalD1999**
**Work Done:**
Added Delete the breakpoint API in debugger-extension API by making a call to the Cloud Debugger Service API. Also added Request Handler in App to make call to the debugger-extension API.

### CODE REVIEW

File: app/src/background/backgroundRequestHandler.ts [hunk 1]
```diff
@@ -23,11 +23,7 @@ export class BackgroundRequestHandler {
...
   static listen() {
     chrome.runtime.onMessage.addListener(
-      (
-        data: backgroundRequest.BackgroundRequestData,
-        sender,
-        sendResponse
-      ) => {
+      (data: backgroundRequest.BackgroundRequestData, sender, sendResponse) => {
         const handler = BackgroundRequestHandler.handlers[data.type];
         if (handler === undefined) {
```
Comment (hananeikhelef) [inline app/src/background/backgroundRequestHandler.ts:28]: Quick questions please, is it mandatory to check this ? ^^

Comment (KushalD1999) [reply app/src/background/backgroundRequestHandler.ts:28]: yes. helps in getting data (required to make API call) and send response back

File: app/src/background/backgroundRequestHandler.ts [hunk 2]
```diff
@@ -101,3 +97,17 @@ BackgroundRequestHandler.on<backgroundRequest.ListBreakpointsData>(
     return response;
   }
 );
+
+/**
+ * Handler for delete the breakpoint from debugger-extension api and return the response.
+ */
+BackgroundRequestHandler.on<backgroundRequest.FetchBreakpointRequestData>(
+  backgroundRequest.BackgroundRequestType.DELETE_BREAKPOINT,
```
Comment (jamohile) [inline app/src/background/backgroundRequestHandler.ts:105]: Not sure if this is in the enum yet, could you add it in if not?

Comment (KushalD1999) [reply app/src/background/backgroundRequestHandler.ts:105]: I will do that.

**Pull request merged**